.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules/
+.DS_Store
```

Comment:
Can you remove this from the project's `.gitignore`? It shouldn't be here, but rather in people's global `~/.gitignore`.